mava/configs/env/gigastep.yaml

```diff
@@ -0,0 +1,11 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: hide_and_seek_5_vs_5 # [hide_and_seek_5_vs_5, identical_5_vs_5, waypoint_5_vs_5]
+
+env_name: Gigastep
+
+eval_metric: win_rate
+
+kwargs:
```

Comment:
The kwargs must be passed to the env initializer (scenario builder), here we can define the max_episode length and all necessary attributes such as obs and action type.